### ISSUE TITLE
ERROR: This package does not have R2PM_INSTALL_WINDOWS instructions

### ISSUE THREAD

**Issue by abasakre** [radareorg/r2ghidra#89]
## Description
ERROR: This package does not have R2PM_INSTALL_WINDOWS instructions

*Please describe what are you missing or wanting to be improved*
ERROR: This package does not have R2PM_INSTALL_WINDOWS instructions
*Provide images, ascii-art, test files and anything that may help us understand your request*
<img width="494" alt="Screenshot_1" src="https://user-images.githubusercontent.com/61644490/184122019-f9259d12-3440-4e69-b397-3a11795aa963.png">

**Comment by trufae** [comment]
Yep that's known and planned to be fixed, first thing was to get r2g to publish the builds in the release page from the CI, which was done in the last week release. next step is to get r2pm to pull those blobs on windows, so we dont need to require users to compile everything

**Comment by abasakre** [comment]
how to fix it ?

**Comment by trufae** [comment]
If you don't wanna wait to get the package fixed you can:

* download the w64 build from the release page of r2ghidra and load it in r2
* Fix the package and submit a PR. you will need to add an R2PM_INSTALL_WINDOWS() function that contain the cmd.exe instructions to pull that .dll and copy it in the right place, see r2 -H for paths

**Comment by kimstik** [comment]
from 5.8.0 error is a bit different:

```
r2pm.exe -ci r2ghidra-sleigh
INFO: Cleaning r2ghidra-sleigh
INFO: Using r2-5.8.0 and r2pm-5.8.0
INFO: Cleaning r2ghidra-sleigh
Cloning into 'r2ghidra-sleigh'...
fatal: unable to access 'https://github.com/radareorg/r2ghidra C:\Users\Cola\.local/share\\radare2\r2pm/git\\r2ghidra-sleigh/': URL using bad/illegal format or missing URL
INFO: Starting install for r2ghidra-sleigh
ERROR: This package does not have R2PM_INSTALL_WINDOWS instructions
```

**Comment by trufae** [comment]
you can manually install the dll and the sleigh files for now. the pkg is still not ready . but ill handle it soon or late

![photo_2023-03-22 13 30 35](https://user-images.githubusercontent.com/6431515/226905388-ed42243d-dd4b-4fd7-8587-4e3ea9119a2c.jpeg)
